### ISSUE TITLE
Sheets not found due to sheets being referenced by absolute path.

### ISSUE THREAD

**Issue by luke-clifton** [qrilka/xlsx#171]
I have an xlsx file that won't parse because:

```
*** Exception: MissingFile "/xl/worksheets/sheet.xml"
CallStack (from HasCallStack):
  error, called at src/Codec/Xlsx/Parser.hs:94:22 in xlsx-1.1.1-054dc7745149d1a0a8c7f44a692517d728629241f887be768ba785b6fb33905d:Codec.Xlsx.Parser
```

That file is in the archive, but it seems that `findEntryByPath` returns `Nothing` with the leading slash.

https://github.com/qrilka/xlsx/blob/426eb0ad3aa6686cfdff309d87a2428df0011f83/src/Codec/Xlsx/Parser.hs#L358

**Comment by qrilka** [comment]
Hi @luke-clifton could you attach a test file maybe? I'd try to find some time over the weekend

**Comment by luke-clifton** [comment]
If I make the below change, it can parse my file. (I've added the `dropWhile (== '/')` in `Codec.Xlsx.Parser`

```
extractSheet ar sst contentTypes caches wf = do
  let filePath = dropWhile (== '/') $ wfPath wf
  file <- note (MissingFile filePath) $ Zip.fromEntry <$> Zip.findEntryByPath filePath ar
```

The file is the Profile.xlsx found in the [Garmin FIT SDK](https://developer.garmin.com/fit/download/)

I've attached it here as well.

[Profile.xlsx](https://github.com/qrilka/xlsx/files/13915308/Profile.xlsx)

**Comment by qrilka** [comment]
Fixed in #172
